src/controls/dynamicForm/IDynamicFormProps.ts

```diff
@@ -13,6 +13,10 @@ export interface IDynamicFormProps {
    * Specifies if the form is disabled
    */
   disabled?: boolean;
+  /**
+   * Specify if save button is disabled.
+   */
+  saveDisabled?: boolean;
```

Comment:
Please update the control's doc page to reflect the new prop added.

Reply:
updated DynamicForm.md with the new prop